server/boot/create-store-mappings.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var debug = require('debug')('boot:create-store-mappings');
+
+var Promise = require('bluebird');
+var _ = require('underscore');
+
+
+module.exports = function(app){
```

Comment:
This can be done better. I have recently realized that boot scripts which work asynchronously should tell the loopback framework when they are done with work via a callback method. Otherwise overtime bugs popup due to the order of execution not being serial in the boot scripts anymore ... it is somewhat similar to the code going ahead and not waiting when we have a broken promise chain.

@aquid has implemented it correctly [here](https://github.com/ShoppinPal/sellerapp/blob/ef5e435b6bca356d04859b62750937a2f181a32a/server/boot/s03-seed-SP-roles.js#L10), please have a look and tweak your code accordingly.